### ISSUE TITLE
Failed to detect internet connection

### ISSUE THREAD

**Issue by mrtshoot** [MortezaBashsiz/nipovpnbash#17]
I setup your nipovpn on my ubuntu 20.04 LTS server and it's located on Hetzner
After setup I gave my Vmess url and set it on v2rayng android application it's successfully connect but when I tap connection and test it on v2rayng I see the following error:

Failed to detect internet connection: io: read/write on closed pipe

**Comment by MortezaBashsiz** [comment]
Which method you are using?
Only External ?
Internal and External ?

**Comment by mrtshoot** [comment]
> Which method you are using? Only External ? Internal and External ?

I used external with arvancloud cdn

**Comment by MortezaBashsiz** [comment]
Please try it:
Enter the A record instead of CNAME in your android client in address field and see if it works or not
This is just for test and do not use it for long time

**Comment by mrtshoot** [comment]
> Please try it: Enter the A record instead of CNAME in your android client in address field and see if it works or not This is just for test and do not use it for long time

I tested it and it's work correctly with A Record

**Comment by MortezaBashsiz** [comment]
So this problem is with your Arvan
Pull the repository again to make sure that you have the latest commit
Run it again

**Comment by mrtshoot** [comment]
Thanks, so I will check it again and come back here to share report

**Comment by mrtshoot** [comment]
Finally, I fixed it.
Something strange happened.
I thought the problem was related to my server location so I changed the server location from Germany to Finland but unfortunately, it didn't connect. so I thought it was about the server IP and I changed it but the problem didn't solve.
In the end, I changed the A Record and CNAME Record from vpn and gheychivpn to another thing and it connected successfully.
